backends/dpdk/dpdkHelpers.cpp

```diff
@@ -765,6 +765,33 @@ bool ConvertStatementToDpdk::preorder(const IR::MethodCallStatement *s) {
             } else {
                 ::error("%1%: unhandled function", s);
             }
+        } else if (a->method->name == "mirror_packet") {
+            auto slot_id = a->expr->arguments->at(0)->expression;
```

Comment:
first you should check that there are exactly 2 arguments and give an error otherwise (probably a modelError).

Reply:
No of arguments mismatch is caught by the frontend, I also added a test for that.
I can add the check to be on the safer side.

Reply:
Added a check.